executor/point_get.go

```diff
@@ -337,6 +337,15 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 
 // EncodeUniqueIndexKey encodes a unique index key.
 func EncodeUniqueIndexKey(ctx sessionctx.Context, tblInfo *model.TableInfo, idxInfo *model.IndexInfo, idxVals []types.Datum, tID int64) (_ []byte, err error) {
+	encodedIdxVals, err := EncodeUniqueIndexValues(ctx, tblInfo, idxInfo, idxVals)
+	if err != nil {
+		return nil, err
+	}
+	return tablecodec.EncodeIndexSeekKey(tID, idxInfo.ID, encodedIdxVals), nil
+}
+
+// EncodeUniqueIndexValues encodes a unique index values.
+func EncodeUniqueIndexValues(ctx sessionctx.Context, tblInfo *model.TableInfo, idxInfo *model.IndexInfo, idxVals []types.Datum) (_ []byte, err error) {
```

Comment:
It would more clear if we rename it to `EncodeUniqueIndexValuesForKey`, so we don't confuse it with methods like `tablecodec.GenIndexValue`.